### PR TITLE
Update user guide for add and delete position

### PR DESCRIPTION
**Pull request by Harry-Gao-H**
Update user guide for add and delete position and also modify the introduction of user guide

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -72,6 +72,29 @@ Shows a message explaning how to access the help page.
...
 Format: `help`
 
+### Adding a position: `add position`
+
+Adds a position to the application management system.
+
+Format: `add position pos/POSITION desc/DESCRPTION`
+
+Examples:
+* `add postion pos/software engineer desc/work in a team that builds a face recognition application`
```
Comment (BenjaminTan99) [inline docs/UserGuide.md:82]: Typo for 'add postion' should be position

File: docs/UserGuide.md [hunk 2]
```diff
@@ -72,6 +72,29 @@ Shows a message explaning how to access the help page.
 
 Format: `help`
 
+### Adding a position: `add position`
+
+Adds a position to the application management system.
+
+Format: `add position pos/POSITION desc/DESCRPTION`
```
Comment (BenjaminTan99) [inline docs/UserGuide.md:79]: Typo for DESCRPTION should be DESCRIPTION

**Review by BenjaminTan99 — CHANGES_REQUESTED** [review]
Some minor errors but LGTM.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -3,14 +3,14 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+MrTechRecruiter (MTR) is a **desktop app for managing job application, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, MTR can get your application management tasks done faster than traditional GUI apps.
```
Comment (LMAOboxhack) [inline docs/UserGuide.md:6]:
```suggestion
MrTechRecruiter (MTR) is a **desktop app for managing job applications, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, MTR can get your application management tasks done faster than traditional GUI apps.
```

File: docs/UserGuide.md (hunk 1 above)
Comment (LMAOboxhack) [reply docs/UserGuide.md:82]:
```suggestion
* `add postion pos/software engineer desc/work in a team that builds a facial recognition application`
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -72,6 +72,29 @@ Shows a message explaning how to access the help page.
...
+
+
+### Deleting a position: `delete position`
+
+Deletes the specified position from the application management system.
+
+Format: `delete position pos/POSITION`
+
+* Deletes the position with the specified `POSITION`.
+* The name refers to the name of the positions as shown in the displayed position list.
```
Comment (LMAOboxhack) [inline docs/UserGuide.md:92]:
```suggestion
Format: `delete position name/NAME`

* Deletes the position with the specified `NAME`.
* The name refers to the name of the position as displayed in the list of positions.
```

Might consider changing `pos/POSITION` to `name/NAME` instead for better clarity? (for `add` as well)
"Deletes the position with the specified `POSITION`" is a little ambiguous.

**Review by LMAOboxhack — COMMENTED** [review]
Some discussions on command syntax & typo nits, otherwise LGTM!

File: docs/UserGuide.md (hunk 4 above)
Comment (Harry-Gao-H) [reply docs/UserGuide.md:92]: Yes `name` is clearer :slightly_smiling_face: Thanks for pointing that out.

**Pull request merged**